api/reviews/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+
+const router = express.Router();
+
+const ProjectReviews = require('../../models/projectReviewsModel');
+
+router.post('/', async (req, res) => {
+    try {
+      const [projectReview] = await ProjectReviews.insert(req.body);
+      res.status(201).json({ projectReview });
+    } catch (error) {
+      console.log(error);
+      res.status(500).json({ error, message: 'Unable to add review' });
+    }
+  });
+
+  router.get('/:id', async (req, res) => {
+    try {
+      const { id } = req.params;
+      const projectReviews = await ProjectReviews.findById(id);
+      if (projectReviews) {
+        res.status(200).json({ projectReviews, msg: "Review was found" });
```

Comment:
**suggestion**: This is just a single review, right? So it should be `projectReview`.